.github/workflows/ci.yml

```diff
@@ -65,3 +65,14 @@ jobs:
              "image": "inspirehep/editor",
              "tag": "${{ steps.describe.outputs.tag }}"}
         if: ${{ github.event_name == 'push' }}
+      - name: deploy on prod
+        uses: peter-evans/repository-dispatch@v1
+        with:
+          repository: inspirehep/kubernetes
+          token: ${{ secrets.INSPIRE_BOT_TOKEN }}
+          event-type: deploy
+          client-payload: >
+            {"environment": "prod",
+             "image": "inspirehep/editor",
+             "tag": "${{ steps.describe.outputs.tag }}"}
+        if: ${{ github.event_name == 'release' }}
```

Comment:
are you sure this will work? it doesn't look like we'll build the image in this case

Reply:
you're right the tagged image will not be there, not sure about the syntax though